src/templates/pycontw-2020/contents/zh/events/warmup-session.html

```diff
@@ -7,13 +7,19 @@
 
 {% block content %}
 {% url 'page' path='events/passion-fruit' as passion_fruit_url %}
+{% url 'page' path='events/taipei-dot-py' as taipei_dot_py %}
+
 
 <section class="pill-title alt">
     <h1>暖身活動</h1>
 </section>
 
-<h3><a href="{{ passion_fruit_url }}">第壹次Python NPF百香果 x 南投.py</a></h3>
+<h3><a href="{{ passion_fruit_url }}">第壹次 Python NPF 百香果 x 南投.py</a></h3>
```

Comment:
If we're adding spaces here, should we also change the title of the content page, located at `src/templates/pycontw-2020/contents/zh/events/passion-fruit.html`, and also the en one.